versioned_docs/version-0.x.0/kubeslice-cli/reference/configuration-parameters.mdx

```diff
@@ -0,0 +1,85 @@
+# Configuration Parameters
+This topic describes the parameters used in creating the topology YAML file to install opensource KubeSlice.
+
+## Cluster Configuration Parameters
+The following table describes the cluster configuration parameters used to create the topology YAML.
+
+|Parameter|Parameter Type|Description|Required|
+|----|----|----|----|
+|profile|String|It is the KubeSlice profile to create demo clusters.The support values are **minimal-demo** and **full-demo**|Mandatory|
+|kube_config_path|String|It is the absolute path to the kubeconfig file to use topology setup; topology only| Optional|
+|[controller](#controller-cluster-configuration-parameters)|Object| It is the controller cluster details.|Mandatory|
+|[worker](#worker-cluster-configuration-parameters)|Object|It is the worker clusters details.|Mandatory|
+
+### Controller Cluster Configuration Parameters
+The following table describes the controller cluster configuration parameters used to create the topology YAML .
+
+|Parameter|Parameter Type|Description|Required|
+|----|----|----|----|
+|name|String|It is the user-defined name of the controller cluster.|Mandatory|
+|context_name|String|It is the name of the context to use from the kubeconfig file; It is used in the topology YAML file only.|Optional|
+|kube_config_path|String|It is the absolute path to the kubeconfig file to install controller cluster. This takes precedence over the kubeconfig path defined under cluster configuration.|Optional|
+|control_plane_address|AlphaNumeric|It is the address of the control plane kube-apiserver. The kubeslice-cli determines the address from kubeconfig. Override this flag if the address in kubeconfig is not reachable by other clusters in topology.|Mandatory|
+|node_ip|Numeric|It is the IP address of one of the node in the cluster. The kubeslice-cli determines the address from this command:`kubectl get nodes`. Override this flag to an address which is discoverable by other clusters in the topology.| Mandatory|
+
+### Worker Cluster Configuration Parameters
+The following table describes the worker cluster configuration parameters.
+
+|Parameter|Parameter Type|Description|Required|
+|----|----|----|----|
+|name|String|It is the user defined name of the worker cluster.|Mandatory|
+|context_name|String|It is the name of the context to use from kubeconfig file; for topology only|Optional|
+|kube_config_path|String|It is the absolute path to the kubeconfig file to install a worker cluster. This takes precedence over the kubeconfig path defined under cluster configuration.|Optional|
+|control_plane_address|AlphaNumeric|It is the address of the control plane `kube-apiserver`. The kubeslice-cli determines the address from kubeconfig. Override this flag if the address in kubeconfig is not reachable by other clusters in topology.|Mandatory|
```

Comment:
Override this option when the address is 

Don't use flag....

Reply:
done